backend/endpoints/collections.py

```diff
@@ -183,12 +184,12 @@ async def update_collection(
 
             artwork_file = artwork.file.read()
             file_location_s = f"{artwork_path}/small.{file_ext}"
-            with open(file_location_s, "wb+") as artwork_s:
+            async with await open_file(file_location_s, "wb+") as artwork_s:
                 artwork_s.write(artwork_file)
                 fs_resource_handler.resize_cover_to_small(file_location_s)
 
             file_location_l = f"{artwork_path}/big.{file_ext}"
-            with open(file_location_l, "wb+") as artwork_l:
+            async with await open_file(file_location_l, "wb+") as artwork_l:
```

Comment:
whats the main benefit of this `async with await` pattern?

Reply:
That's just how the library has `open_file` implemented, so `await open_file` returns the async context manager, that can be used with `async with`!